### ISSUE TITLE
Improve Dashboard Speed

### ISSUE THREAD

**Issue by driaug** [useplunk/plunk#41]
Currently, there are three parts of the dashboard that don't stay fast at scale

These are:
- The feed on the main page of the dashboard
- The analytics page
- The campaign page (due to the loading of the contact selector)

There is a lot of improvement that can be done here!

**Comment by killion** [comment]
Yeah, I've been hitting timeouts with the campaign page. Our newsletters are over 500K emails. Maybe removing the count or having a special `COUNT(*)` to get the numbers instead?

**Comment by yasoob** [comment]
@killion this is a bit off topic but considering that you mentioned your newsletter subscriber count, do you mind sharing what was your system resource usage while sending a campaign to your whole list and what was the performance like? (how long did it take to deliver the whole campaign)

**Comment by killion** [comment]
System utilization is low during the actual sends. The system can lock when loading the pages with counts on them though. I ran into a problem when I would set the number of emails per batch higher that duplicate jobs would run and send the same email multiple times. I added a PR for locking but I haven't been able to get back and finish working on it.
